samples/event_manager_proxy/README.rst

```diff
@@ -0,0 +1,151 @@
+.. _event_manager_proxy_sample:
+
+Event Manager proxy
+###################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Event Manager proxy sample demonstrates the functionality of :ref:`event_manager_proxy` to transport events between remote and host cores.
+The sample also demonstrates the proposed application structure where common events declarations and definitions are available for both cores.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf5340dk_nrf5340_cpuapp
+
+Overview
+********
+
+Event Manager proxy sample copies the functionality from :ref:`app_event_manager_sample` sample but splits it between the two cores.
+On the remote core, we have functionality that relates to the simulated sensor, while the host core take care of configuration and calculates statistics.
+
+The sample copies modules from :ref:`app_event_manager_sample` sample:
+
+.. include:: ../app_event_manager/README.rst
+   :start-after: event_manager_sample_modules_start
+   :end-before: event_manager_sample_modules_end
+
+The only change is that the Statistic module now also counts control messages.
+The change was implemented to transfer different range of messages between cores.
+
+Files and directories layout
+============================
+
+The application is divided into host and remote.
+The host configuration is placed directly in the :file:`samples/event_manager_proxy` folder.
+The remote configuration is placed in the :file:`samples/event_manager_proxy/remote` folder.
+The remote configuration is added to the host as a subproject which means that it builts as part of the host application build.
+
+Both remote and host use common event declarations and definitions that are placed in the :file:`samples/event_manager_proxy/common_events` folder.
+Both use modules that are placed in the :file:`samples/event_manager_proxy/modules` folder.
+
+Configuration
+*************
+
+|config|
```

Comment:
No custom configuration described, so this optional section can be removed, as per template.